database/src/adapters/aggregated/circulating_supply.rs

```diff
@@ -6,7 +6,7 @@ use tracing::error;
 use crate::models::aggregated::circulating_supply::CirculatingSupply;
 use crate::schema;
 
-pub(crate) async fn add_circulating_supply(
+pub async fn add_circulating_supply(
```

Comment:
Here and in all other places, it's better to use the narrowest scope which you can use. 
It's usually `pub(crate)`